packages/primitives/src/FileUpload.tsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Assign, FileUpload, FileUploadHiddenInput, fileUploadAnatomy } from "@ark-ui/react";
+import { RecipeVariantProps, sva } from "@ndla/styled-system/css";
+import { JsxStyleProps } from "@ndla/styled-system/types";
+import { createStyleContext } from "./createStyleContext";
+import { Label } from "./Label";
+import { TextProps, Text } from "./Text";
+
+const fileUploadRecipe = sva({
+  slots: fileUploadAnatomy.keys(),
+  base: {
+    root: {
+      display: "flex",
+      flexDirection: "column",
+      minHeight: "100%",
+    },
+    dropzone: {
+      display: "flex",
+      gap: "small",
+      flexDirection: "column",
+      alignItems: "center",
+      justifyContent: "center",
+      padding: "xxlarge",
+      textStyle: "label.medium",
+      marginBottom: "medium",
+      borderRadius: "xsmall",
+      border: "1px solid",
+      borderColor: "stroke.subtle",
+      _hover: {
+        borderStyle: "dashed",
+      },
+    },
+    itemGroup: {
+      display: "flex",
+      flexDirection: "column",
+      gap: "xsmall",
+    },
+    item: {
+      width: "100%",
+      display: "grid",
+      gridTemplateAreas: '"preview name delete"\n "preview size delete"',
```

Comment:
Trenger vel ikke en newline her. Se `Controls.tsx` for å se hvordan det kan gjøres.

Reply:
Dette gjelder vel fortsatt.